ado/ipacheckreadreplace.ado

```diff
@@ -83,16 +93,31 @@ qui {
 		di as error "selectmultiple specified for `r(N)' corrections entries without oldvalue"
 		error 498
 	}
-	cap assert missing(`oldvalue') if missing(`selectmultiple')
-	if _rc {
-		count if !missing(`oldvalue') & missing(`selectmultiple')
-		di as error "oldvalue specified for `r(N)' corrections entries without selectmultiple"
-		error 498
-	}
 	
-	tempfile selectmult allcorrections
+	***************** remove entries that should be dropped ********************
+	tempfile selectmult allcorrections droplist
 	save `allcorrections'
-
+	if !missing("`drop'") {
+		cap tostring `drop', replace
+		replace `drop' = lower(strtrim(`drop'))
```

Comment:
My main concern is that someone might confuse columns and accidentally drop something that they didn't mean to drop. Should the relative ease of "yes" or "1" take precedent over the accidental drop protection of "drop"? I could see merits to either one.